src/ez/Table.java

```diff
@@ -98,6 +102,29 @@ public Table varchar(String name, int n) {
     return column(name, "VARCHAR(" + n + ")");
   }
 
+  public Table linkColumn(String name, Class<?> type, String foreignTable, String foreignColumnName,
+      String foreignKeyName) {
```

Comment:
lets just not let people specify foreignKeyName

Reply:
removed the link column with foreignKeyName